.github/workflows/build.yml

```diff
@@ -69,11 +71,76 @@ jobs:
           version: tags/20240511.0
           file: firebase-windows-amd64.zip
 
-      - run: Expand-Archive -Path firebase-windows-amd64.zip -DestinationPath third_party; Rename-Item -Path third_party/firebase -NewName firebase-development
+      - run: |
+          Expand-Archive -Path firebase-windows-amd64.zip -DestinationPath third_party
+          Rename-Item -Path third_party/firebase -NewName firebase-development
         shell: powershell
 
       - name: Configure
-        run: cmake -B out -D CMAKE_BUILD_TYPE=Release -G Ninja -S ${{ github.workspace }} -D CMAKE_Swift_FLAGS="${{ matrix.options }}" -D SWIFT_FIREBASE_BUILD_EXAMPLES=NO
+        run: |
+          cmake -B out `
+                -D CMAKE_BUILD_TYPE=Release `
+                -G Ninja -S ${{ github.workspace }} `
+                -D CMAKE_Swift_FLAGS="${{ matrix.options }}" `
+                -D SWIFT_FIREBASE_BUILD_EXAMPLES=NO
+
+      - name: Build
+        run: cmake --build out
+
+  android:
+    runs-on: windows-latest
```

Comment:
Dumb question, does Android need to run on a Windows host? It seems like it could/should run on whatever host machine. I guess this is the first time we're building for a platform we're not directly running on. LMK

Reply:
We _could_ get away with another platform, but we want to prioritise Windows. The reason for this is that the DevEx work that we have done is on Windows. We are going to be polishing the experience on Windows and other OSes are going to be a "best effort" with low priority.